tools/Makefile

```diff
@@ -392,6 +392,23 @@ install::
 	  "$(INSTALL_BINDIR)/"
 endif
 
+CAMLTEX= ../compilerlibs/ocamlcommon.cma \
+	../compilerlibs/ocamlbytecomp.cma \
+	../compilerlibs/ocamltoplevel.cma \
+	../otherlibs/str/str.cma \
+	../otherlibs/unix/unix.cma \
+	caml_tex.ml
+
+#Scan latex files, and run ocaml code examples
+
+caml-tex: INCLUDES+= -I ../otherlibs/str -I ../otherlibs/unix
+caml-tex: $(CAMLTEX)
+	$(CAMLC) $(LINKFLAGS) -I .. -linkall -o $@ $(CAMLTEX)
+
+opt.opt:caml-tex
```

Comment:
Why is this target named `opt.opt`? I would expect maybe `all:`, but not `opt.opt` here.

Reply:
This is due to conflicting dependencies:  caml-tex depends on  `Str` and `Unix` while the otherlibraries target depends on the ocamltools target.

Reply:
Could you maybe have a comment to explain this tricky subtlety?

Reply:
Good point, I added a comment.